protocol/communities/manager.go

```diff
@@ -3993,6 +3996,10 @@ func (m *Manager) GetByID(id []byte) (*Community, error) {
 	return community, nil
 }
 
+func (m *Manager) GetByIDReadonly(id []byte) (ReadonlyCommunity, error) {
```

Comment:
Can you please add some func description for both `GetByIDReadonly`  and `GetByID`?
And perhaps mention that `GetByIDReadonly` must be used where possible